intern/service/boss/tests/test_baseversion.py

```diff
@@ -269,7 +269,7 @@ def test_get_user_request_with_email(self):
         actual = self.test_project.get_user_request(
             'POST', 'application/json', url_prefix, token, user, email=email)
 
-    def test_get_user_request_with_password(self):
+    def test_get_user_request_with_password_and_email(self):
```

Comment:
I don't see any use of email in this test?

Reply:
Maybe you meant to rename the test on line 289?

Reply:
aha! yes indeed — fixing. good eye :)